ganttproject/src/main/java/net/sourceforge/ganttproject/storage/SqlProjectDatabaseImpl.kt

```diff
@@ -69,17 +73,81 @@ class SqlProjectDatabaseImpl(private val dataSource: DataSource): ProjectDatabas
       .column(TASK.ID, INTEGER)
       .column(TASK.NAME, VARCHAR.notNull())
       .column(TASK.COLOR, VARCHAR.null_())
+      .column(TASK.SHAPE, VARCHAR.null_())
+      .column(TASK.IS_MILESTONE, BOOLEAN.notNull())
+      .column(TASK.IS_PROJECT_TASK, BOOLEAN.notNull())
+      .column(TASK.START_DATE, TIMESTAMP.notNull())
+      .column(TASK.DURATION, INTEGER.notNull())
+      .column(TASK.COMPLETION, INTEGER.null_())
+      .column(TASK.EARLIEST_START_DATE, TIMESTAMP.null_())
+      .column(TASK.THIRD_DATE_CONSTRAINT, INTEGER.null_())
+      .column(TASK.PRIORITY, VARCHAR.null_())
+      .column(TASK.WEB_LINK, VARCHAR.null_())
+      .column(TASK.COST_MANUAL_VALUE, VARCHAR.null_())
+      .column(TASK.IS_COST_CALCULATED, BOOLEAN.null_())
+      .column(TASK.NOTES, VARCHAR.null_())
       .constraints(primaryKey(TASK.ID))
       .execute()
+
+    dsl.createTable(TASKDEPENDENCY)
```

Comment:
Any sense to use same schema defined in `init-project-database.sql` instead of duplicating it?